README.md

```diff
@@ -47,3 +47,16 @@ rm('foo.txt');
 ```
 
 And then calling it with `node scripts/scriptname.js`
+
+Installing from GitHub?
+----------------------
+
+If you are installing directly from GitHub, `npm install shelljs/shx`, you'll need to build `shx` locally.  This project does not commit built assets to the GitHub repo.  To build locally:
+
+```
+cd node_modules/shx
```

Comment:
How does the `node_modules/shx` directory get created? We should include that command above this. Here's what I tried (that didn't work):

```bash
$ npm install shelljs/shx # I'm expecting this to have errors
... Lots of errors ...
$ cd node_modules/shx/ # I wasn't expecting this command to fail
no such file or directory: node_modules/shx/
```